utils/dd_to_csv.py

```diff
@@ -182,7 +202,9 @@ def convert_dd_to_tabular(basedir: str, output_dir: str) -> None:
 
     # Extract headers with key=param_name and value=List[attributes]
     lines = list(open("xl2times/config/times_mapping.txt", "r"))
-    headers_data = dict()
+    headers_data = headers_by_attr
+    # The following will overwrite data obtained from headers_by_attr
+    # TODO: Remove once migration is done?
```

Comment:
Should we change this code to use the header data from `times-info.json` instead?

Reply:
Do you mean exclusively? Currently times-info.json does allow specifying all the info that is included in `times_mapping.txt`. That's why I kept it for now.

Reply:
Ah I read the code hastily yesterday and thought it was overwriting everything in `headers_by_attr`. I now see that it only overwrites the few mappings that remain in the `times_mapping.txt`.